shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -103,6 +103,7 @@ const NODE_TOTAL = {
 };
 const CLUSTER_AGENT_CUSTOMIZATION = 'clusterAgentDeploymentCustomization';
 const FLEET_AGENT_CUSTOMIZATION = 'fleetAgentDeploymentCustomization';
+const CONTEXT_HOOK_EDIT_YAML = 'show-preview-yaml';
```

Comment:
Sorry to be a PITA, wasn't clear from my example, but this should be an exported property in CruResource (or somewhere sensible and common).

In this file it's imported and used within the saveMachinePools hook. 

Let me know if you want to do that minor refactor. i need to check out and test anyway

Reply:
Not an issue, whatsoever 🙂 

I wasn't finding a common location that I originally liked, and I'm fine with exporting the constant from CruResource.